sdk/src/main/scala/io/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -324,6 +324,19 @@ class AccountForgeMessageBuilder(
       logsBloom
     )
 
+    if (isWithdrawalEpochLastBlock) {
```

Comment:
I'm thinking to put this info in the AccountState.
Currently this log data will be available only in a single Forger node.
What do you think?